aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/DeleteHandler.java

```diff
@@ -24,44 +23,43 @@
         final Logger logger) {
 
         this.logger = logger;
+        ResourceModel model = request.getDesiredResourceState();
+        // STEP 1.0 [initialize a proxy context]
+        ProgressEvent<ResourceModel, CallbackContext> test = proxy
```

Comment:
nit, i'd call this something other than test.

Reply:
oops, I was supposed to change this to something meaningful..